src/Apollo_Client/core/Apollo_Client__Core_WatchQueryOptions.re

```diff
@@ -0,0 +1,32 @@
+module ErrorPolicy = {
+  type t =
+    | None
+    | Ignore
+    | All;
+
+  let toRaw =
+    fun
+    | None => "none"
+    | Ignore => "ignore"
+    | All => "all";
+};
+
+module WatchQueryFetchPolicy = {
+  type t =
+    | CacheAndNetwork
+    // ...extends FetchPolicy
```

Comment:
what this means?

Reply:
It's just saying that in typescript `WatchQueryFetchPolicy` only defines one enum and extends `FetchPolicy`.  I usually try to keep properties in alphabetical order, but here I was experimenting to see if it's more useful to separate each group by what is extended. Maybe we should ignore it or make it clear what the comment is saying 😆